bank/static/css/footer.css

```diff
@@ -1,157 +1,201 @@
-.full {
-    width: 100%;
+@import url(https://fonts.googleapis.com/css?family=Lato:400,300,700,900|Open+Sans:400,300);
+
+footer {
+    background-image: url(webenlance.in/bootsnipp/images/footer_sli2.png);
+    background-repeat: repeat;
+}
+
+.footer_top {
+    min-height: 100px;
+    background: rgba(255, 255, 255, 0);
+    background: -moz-linear-gradient(left, rgba(255, 255, 255, 0) 0%, rgba(247, 247, 247, 0) 56%, rgba(247, 247, 247, 1) 57%, rgba(38, 12, 77, 1) 57%, rgba(38, 12, 77, 0.97) 100%);
+    background: -webkit-gradient(left top, right top, color-stop(0%, rgba(255, 255, 255, 0)), color-stop(56%, rgba(247, 247, 247, 0)), color-stop(57%, rgba(247, 247, 247, 1)), color-stop(57%, rgba(38, 12, 77, 1)), color-stop(100%, rgba(38, 12, 77, 0.97)));
+    background: -webkit-linear-gradient(left, rgba(255, 255, 255, 0) 0%, rgba(247, 247, 247, 0) 56%, rgba(247, 247, 247, 1) 57%, rgba(38, 12, 77, 1) 57%, rgba(38, 12, 77, 0.97) 100%);
+    background: -o-linear-gradient(left, rgba(255, 255, 255, 0) 0%, rgba(247, 247, 247, 0) 56%, rgba(247, 247, 247, 1) 57%, rgba(38, 12, 77, 1) 57%, rgba(38, 12, 77, 0.97) 100%);
+    background: -ms-linear-gradient(left, rgba(255, 255, 255, 0) 0%, rgba(247, 247, 247, 0) 56%, rgba(247, 247, 247, 1) 57%, rgba(38, 12, 77, 1) 57%, rgba(38, 12, 77, 0.97) 100%);
+    background: linear-gradient(to right, rgba(255, 255, 255, 0) 0%, rgba(247, 247, 247, 0) 56%, rgba(247, 247, 247, 1) 57%, rgba(38, 12, 77, 1) 57%, rgba(38, 12, 77, 0.97) 100%);
+    filter: progid:DXImageTransform.Microsoft.gradient(startColorstr='#ffffff', endColorstr='#260c4d', GradientType=1);
+}
+
+.bagcblue {
+    background-color: #260c4d;
+    background-image: url(http://webenlance.in/bootsnipp/images/footer_sli.png);
+    background-repeat: no-repeat;
+    background-position: left top;
+    padding-left: 65px;
+    padding-top: 15px;
+    color: #FFF;
+    font-family: 'Lato', sans-serif;
+    min-height: 131px;
+}
+
+.bagcblue p {
+    font-size: 13px;
 }
 
-.gap {
-    height: 30px;
-    width: 100%;
-    clear: both;
-    display: block;
+.bagcgrey {
+    background-image: url(http://webenlance.in/bootsnipp/images/footer_sli2.png);
```

Comment:
Cant we have an image of our own for this? Or is this just some template we are editing?